src/ui/molecules/footer/footer.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../styles/themes.scss';
+
+.okp4-footer-main {
+  @include with-theme() {
+    background: themed(footer-background);
+    width: calc(100% - 80px);
+    padding: 45px 20px 45px 60px;
```

Comment:
On large screen, we have a left padding which is 60px instead of 45px, is there any reason? 
Because on other screen sizes, both sides are well balanced.

Reply:
The style follows the mock-up